packages/pigment-css-unplugin/src/index.ts

```diff
@@ -134,6 +138,22 @@ export const plugin = createUnplugin<PigmentOptions, true>((options) => {
       };
     },
   };
+
+  let webpackResolver: AsyncResolver;
+
+  const asyncResolve: AsyncResolver = async (what, importer, stack) => {
+    const result = asyncResolveOpt?.(what);
+    if (typeof result === 'string') {
+      return result;
+    }
+    // Use Webpack's resolver to resolve actual path but
+    // ignore next.js files during evaluation phase of WyW
```

Comment:
Not related to this PR but it seems like we can ignore the `emotion` as well for the `.toString` error, right?

Reply:
Let me check if it fixes emotion related issue but I am sure these two are unrelated. Maybe we can point that emotion import to a dummy file similar to what we do for `next/image`/`next/font`.